tools/certa/src/runners/chrome/webserver.ts

```diff
@@ -52,9 +54,34 @@ app.use("*", (req, resp) => {
   resp.sendStatus(404);
 });
 
-// Run the server...
-app.set("port", process.env.CERTA_PORT || 3000);
-app.listen(app.get("port"), () => {
-  console.log(`Certa web frontend now listening on port ${app.get("port")}`);
-  process.send!("Ready");
+// Once the server actually starts, we should log and send the actual port back to the parent process
+let port = parseInt(process.env.CERTA_PORT ?? "3000", 10);
```

Comment:
Any reason to not check if the port is currently occupied before starting the server initially?  Right now it just looks like it will fallback to the error handling even if we could detect the port's in use.

Reply:
I mean, that's really how `detect` works under the hood anyway, so I figured we might as well just make one attempt with the preferred port anyway if we're going to retry...